gulpfile.js

```diff
@@ -15,10 +15,19 @@
  */
 
 const gulp = require('gulp');
+const gutil = require('gulp-util');
 const Karma = require('karma').Server;
 const karmaDefault = require('./karma.conf');
 
+
 gulp.task('test', function(done) {
+   try {
+      require('./readiness-tool/vendors.json');
+      console.log("vendors.json was found with no problems");
+    } catch (err){
+      gutil.log(gutil.colors.red('vendors.json failed to load because this error: ' + err.message));
+      return;
+    }
```

Comment:
Doing this in the test causes a bundle error when the vendors.json file has extra brackets and commas. I can catch the error in the gulpfile but the test wasn't allowing me to.